lasio/reader.py

```diff
@@ -323,6 +323,10 @@ def determine_section_type(section_title):
         return "Data"
     elif stitle[:2] == "~O":
         return "Header (other)"
+    # las30 transitional code till data parsing is robust for ~A and '_Data'
+    # sections
+    elif stitle.find('_Data') > -1:
```

Comment:
I'm being really picky here but ``> -1`` feels like C - can we use ``elif re.search('_Data', stitle):`` instead? Feels more Pythonic?

Reply:
It's a good point, I'll make the change.